DNN Platform/Library/Entities/Modules/Settings/SettingsRepository.cs

```diff
@@ -63,23 +63,16 @@ public void SaveSettings(ModuleInfo moduleContext, T settings)
 
                         if(string.IsNullOrEmpty(settingValueAsString))
                         {
-                            if (settingValue is DateTime)
-                            {
-                                settingValueAsString = ((DateTime)settingValue).ToString("u");
-                            }
-                            else
-                            {
-                                settingValueAsString = settingValue.ToString();
-                            }
+                            settingValueAsString = GetSettingValueAsString(settingValue);
```

Comment:
So your concern is with code trying to deserialize a date outside of the `SettingsRepository`?  Sounds reasonable, just wanting to make sure I've got the scope of your concern.  `Convert.ToString` with `InvariantCulture` would be fine if everything went through the `SettingsRepository` during retrieval, yes?